src/L2VoteAggregator.sol

```diff
@@ -68,7 +83,11 @@ abstract contract L2VoteAggregator {
 
   /// @dev Emitted when a vote is cast on L2.
   event VoteCast(
-    address indexed voter, uint256 indexed proposalId, VoteType support, uint256 weight
+    address indexed voter,
+    uint256 indexed proposalId,
```

Comment:
we may have to remove `indexed` from proposalId to comport w/ Tally event

Reply:
Should we also make this change in the L1VotePool?

Reply:
I am guessing not because the underlying Governor should emit a compatible event correct?

Reply:
Yeah it shouldn't matter on the L1, since that won't be added to Tally or other tooling. It's really just on the L2 where we want Tally to treat the pool like a Governor.